### PR TITLE
Update maintained state

### PR DESCRIPTION
**Pull request by alexAubin**
Another update from the maintenance ping mechanism

### CODE REVIEW

**Comment by kay0u** [comment]
Could be awesome if changes were made on apps.json and not community, but thanks! (I have no idea which type of black magic is used for the **maintenance ping mechanism**)

File: community.json [hunk 1]
```diff
@@ -22,6 +22,12 @@
         "state": "inprogress",
         "url": "https://github.com/YunoHost-Apps/abantecart_ynh"
     },
+    "adguard": {
+        "branch": "master",
+        "revision": "HEAD",
+        "state": "working",
```
Comment (kay0u) [inline community.json]: https://github.com/YunoHost-Apps/adguard_ynh/issues/2

**Comment by alexAubin** [comment]
>Could be awesome if changes were made on apps.json and not community

Omg yes indeed :sweat_smile: Thanks for noticing xD

I'll fix the script

The whole thing is done from here btw : https://github.com/YunoHost/tartiflette/tree/master/app/scripts/maintenancePing (didnt know where else to put it)

**Comment by kay0u** [comment]
Thank you!

**Pull request closed without merging**